src/api/queries/videos.ts

```diff
@@ -85,7 +85,7 @@ export const GET_NEWEST_VIDEOS = gql`
 
 export const GET_FEATURED_VIDEOS = gql`
   query GetFeaturedVideos {
-    featuredVideos(orderBy: createdAt_DESC) {
+    featuredVideos {
```

Comment:
We still need that argument. If mocking code returns incorrect results because of it, we need to change the mocking resolver to strip that argument instead of removing it from the query